lte/gateway/c/core/BUILD.bazel

```diff
@@ -1185,7 +1185,7 @@ cc_binary(
     srcs = ["oai/oai_mme/oai_mme.c"],
     copts = ["-DEMBEDDED_SGW=1"],
     linkstatic = True,
-    tags = TAG_SERVICE,
+    tags = TAG_SERVICE + ["no-cache"],
```

Comment:
What is the difference in local build times?
I.e. the difference between these scenarios:

1.:
- Have `tags = TAG_SERVICE` (i.e. no change)
- Empty cache completely
- Build `agw_of`
- `bazel clean`
- Build `agw_of` again and note the build time

2.:
- Have `tags = TAG_SERVICE + ["no-cache"],`
- Empty cache completely
- Build `agw_of`
- `bazel clean`
- Build `agw_of` again and note the build time

Reply:
The CI runs with remote cache do not give any information on this since the artifact is already cached, even if we set the no-cache attribute now.

Reply:
The runtime for scenario 1: 5.176s
The runtime for scenario 2: 5.961s

Reply:
Okay that seems reasonable, thanks for the data!